sshproxy.go

```diff
@@ -30,7 +31,7 @@ import (
 	"os/exec"
 	"path/filepath"
 
-	"github.com/kr/pty"
+	"github.com/resin-io-playground/gexpect/pty"
```

Comment:
I needed [this change](https://github.com/resin-io-playground/gexpect/commit/5351e958d5ed8bc7db68873b7d93ea2d2c60f65d), will move to `resin-io-modules`.